src/CycloneDX.Utils/Merge.cs

```diff
@@ -230,6 +236,22 @@ public static Bom HierarchicalMerge(IEnumerable<Bom> boms, Component bomSubject)
                 {
                     result.Metadata.Tools.Tools.AddRange(bom.Metadata.Tools.Tools);
                 }
+                if (bom.Metadata?.Tools?.Components?.Count > 0)

```

Comment:
How should components/services in the tools be handled?
Should they be flat merged or namespaced?

Reply:
What I mean is the following:
Let's say you have this tool in both BOMs that you merge:
```
{
      "bom-ref":"tool1",
      "type": "application",
      "name": "tool1",
      "version": "1"
}
```
What should be the resulting tools?
What meant with "flat merge" is: there would be a single tool exactly as above.
What meant with "namespaced" is: there would be two tools, one with a bom-ref "thing1@tool1" and the other with "thing2@tool1" (if thing1 and thing2 are the main components of the two merged BOMs).

Reply:
(The problem with the flat merge is that if two different tools happen to use the same bom-ref, this will invalidate the uniqueness of the bom-ref in the merged BOM.)

Reply:
As I understand it, the components themselves are also namespaces in hierarchical merges, so it makes only sense to apply the same logic to tool-components.

Reply:
I have adapted it.